servicecontrol/troubleshooting.md

```diff
@@ -105,3 +105,11 @@ The internal *FailedMessageRetries* collection must be purged in order to restor
 1. Select the "FailedMessageRetries" collection in the left tree
 1. Delete all documents in the collection
 1. Stop maintenance mode
+
+## SmartScreen blocks the installer
+
+The installer is [code signed](https://en.wikipedia.org/wiki/Code_signing), but [SmartScreen](https://en.wikipedia.org/wiki/Microsoft_SmartScreen) (called Windows SmartScreen, Windows Defender SmartScreen and SmartScreen Filter in different places) may classify the code signing certificate as "untrusted" and block the installer from running until permission is granted by the user. The blocking of SmartScreen is because the publisher is untrusted. All artifacts during builds are virus scanned before packaging to reduce any potentiaon inclusion of malware in our installers.
+
+Although the installer is code signed correctly with a certificate owned by "NServiceBus Ltd", SmartScreen will block it from running until Microsoft has built enough "trust" in the certificate. One of the main inputs to building that trust is when users grant permission to run the installer. To grant permission to run the installer, click "Run Anyway". This will no longer be required when Microsoft decides to trust the certificate.
+
+
```

Comment:
Lead with the why the code signing cert might be temporarily untrusted, not with a bombastic statement "because teh publisher is untrusted".

This also removes the word "our" which is what content verification is upset about.

```suggestion
The installer is [code signed](https://en.wikipedia.org/wiki/Code_signing), but [SmartScreen](https://en.wikipedia.org/wiki/Microsoft_SmartScreen) (called Windows SmartScreen, Windows Defender SmartScreen and SmartScreen Filter in different places) may classify the code signing certificate as "untrusted" and block the installer from running until permission is granted by the user.

Although the installer is code signed correctly with a certificate owned by "NServiceBus Ltd", SmartScreen will block it from running until Microsoft has built enough "trust" in the certificate. One of the main inputs to building that trust is when users grant permission to run the installer. To grant permission to run the installer, click "Run Anyway". This will no longer be required when Microsoft decides to trust the certificate.

When building ServiceControl, all build artifacts are virus scanned to ensure no viruses or malware are shipped with the installer packages.
```